### DISCUSSION TITLE
On Ubuntu 23.04, running profile does not output anything to the console.

### DISCUSSION THREAD

**Discussion by havefuncoding1** [libbpf/libbpf-bootstrap#239] [Q&A]
Hi, I was trying [profile](https://github.com/libbpf/libbpf-bootstrap/blob/master/examples/c/profile.c) on Ubuntu 23.04, and there was no output from the program. I know this is due to event_handler not getting invoked. Does anyone know why?
```
static int event_handler(void *_ctx, void *data, size_t size)
```

Also, I copied profile binaries to 20.04 and then execution failed with below. Is this expected? I am curious if this could be worked around with different Makefile flags. Thanks a lot in advance for your help.
```
./profile: /lib/aarch64-linux-gnu/libc.so.6: version `GLIBC_2.32' not found (required by ./profile)
./profile: /lib/aarch64-linux-gnu/libc.so.6: version `GLIBC_2.33' not found (required by ./profile)
./profile: /lib/aarch64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by ./profile)
```

**Comment by havefuncoding1** [comment]
Hi @anakryiko do you by any chance know what might be going wrong here? Thanks.

**Reply by anakryiko** [reply]
`version `GLIBC_2.32' not found ` is due to you compiling program on the system with new glibc, and then trying to run it in a different environment with older glibc. Either downgrade glibc on build machine, or upgrade it on target machine. Or try doing static linking to avoid shared library dependencies.
